cli/popper/commands/cmd_init.py

```diff
@@ -10,13 +10,13 @@
 def cli(ctx):
     """Initializes a repository by creating the .popper.yml file.
     """
-    if pu.is_popperized():
+    if pu.is_popperized(scm.get_root_folder()):
```

Comment:
there's a `project_root` variable below that can be used to avoid having to call this function multiple times

Reply:
@ivotron  Please have a look